docs_generator.py

```diff
@@ -41,8 +41,7 @@ def print_functions(baseclass):
             baseclass, predicate=inspect.ismethod):
         if func_name is not '__init__' and not func_name.startswith('_'):
             func = getattr(baseclass, func_name)
-            print doc2md.doc2md(func.__doc__,
-                make_function_name(func, func_name), type=1)
+            print doc2md.doc2md(func.__doc__, make_function_name(func, func_name), type=1)
```

Comment:
Let's use the `print()` function instead of the `print` expression -- it's forwards-compatible.